help/en/package/jmri/jmrit/beantable/LogixNGModuleTable.shtml

```diff
@@ -115,10 +115,12 @@
       <dl>
         <dt>Tools</dt>
 
-        <dd>The tools menu has one item: <strong>Open PickList Tables</strong>. The PickList was
-        designed for dragging names, such as sensor names, to a text field. Due to the
-        implementation of ConditionalNG edit dialogs, the drag and drop capability does not
-        function. It can still be used as a reference.</dd>
+        <dd>The tools menu has two items: <strong>Open clipboard</strong> and <strong>Open PickList
+        Tables</strong>.  See <a href="../../../../html/tools/logixng/reference/chapter2.shtml">
+        LogixNG Reference - Chapter 2</a> for details on the using the clipboard. The PickList was
+        designed for dragging names, such as sensor names, to a text field. Due to the implementation
+        of ConditionalNG edit dialogs, the drag and drop capability does not function. It can still
+        be used as a reference.</dd>
```

Comment:
This is true. But with one exception: If you are writing a formula, you can drag and drop from the PickList to the formula. It seems that all the JEditField's support drag and drop from PickList.

If it's desired to drag and drop from the PickList to ConditionalNG edit dialogs, it should be possible to fix. The dialogs uses BeanSelectPanel so if BeanSelectPanel is updated to handle drag and drop from the PickList, every action and expression that uses BeanSelectPanel will be updated at once. See: jmri.util.swing.BeanSelectPanel
If this is desired, it would be good to open an issue for it.

Reply:
I have updated the PickList comment.  Personally, I dislike the PickList.  I find it cumbersome to use.